java/src/jmri/managers/DefaultVariableLightManager.java

```diff
@@ -0,0 +1,127 @@
+package jmri.managers;
+
+import javax.annotation.*;
+import java.beans.PropertyChangeEvent;
+
+import jmri.InstanceManager;
+import jmri.Light;
+import jmri.LightManager;
+import jmri.Manager;
+import jmri.VariableLight;
+import jmri.VariableLightManager;
+import jmri.SystemConnectionMemo;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Abstract partial implementation of a VariableLightManager.
```

Comment:
No reason to block, but is this statement correct given that this is a concrete class, not an abstract one?

Reply:
Thanks! Copy/paste-error.